pkg/apiclient/offlineclient.go

```diff
@@ -0,0 +1,49 @@
+package apiclient
```

Comment:
minor `offline_client.go`

Reply:
should it be `-` to be consistent with `argo-kube-client.go` etc?

Reply:
I don't feel strongly about it